packages/x-tailwindcss/src/x-tailwind-plugin/components/filter/sizes.ts

```diff
@@ -0,0 +1,29 @@
+import { TailwindHelpers } from '../../../types';
+
+/**
+ * Returns the `sizes` variants for component `filter`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+export function filterSizes(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+
+  return {
+    sm: {
+      minHeight: theme('spacing.24'),
+      gap: theme('spacing.2'),
+      fontSize: theme('fontSize.sm')
+    },
+    md: {
+      minHeight: theme('spacing.32'),
+      gap: theme('spacing.4'),
+      fontSize: theme('fontSize.sm')
+    },
+    lg: {
+      minHeight: theme('spacing.48'),
+      gap: theme('spacing.12'),
+      fontSize: theme('fontSize.md')
```

Comment:
where did you get this value? I cannot find it

Reply:
That wasn't specified in the properties, but was applied in the filter example.
Cris will include it in Figma 👌